itou/www/apply/views/submit_views.py

```diff
@@ -80,12 +80,28 @@ def __init__(self):
         self.company = None
         self.apply_session = None
         self.hire_process = None
+        self.prescription_process = None
+        self.auto_prescription_process = None
         self.is_gps = False
 
     def setup(self, request, *args, **kwargs):
         self.company = get_object_or_404(Company.objects.with_has_active_members(), pk=kwargs["company_pk"])
         self.apply_session = SessionNamespace(request.session, f"job_application-{self.company.pk}")
         self.hire_process = kwargs.pop("hire_process", False)
+        self.prescription_process = (
+            not self.hire_process
+            and request.user.is_authenticated
+            and (
+                request.user.is_prescriber
+                or (request.user.is_employer and self.company != request.current_organization)
```

Comment:
Je pense que l'on aura (comme toujours) des soucis avec la `request.current_organization` (un employeur avec 2 structures fait une prescription dans son autre structure au lieu d'une auto-prescription).
Je me demande si on ne devrait pas plutôt vérifier `self.company not in requests.organizations` ?

Reply:
Effectivement il faudrait changer d'autres choses dans le tunnel pour faire cela. Donc il semble effectivement plus sage de rester sur la `current_organization`.